package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "trackovid",
+  "description": "Landing page for Trackovid project",
+  "version": "1.0.0",
+  "author": "Tech4covid19",
+  "homepage": "https://github.com/Tech4covid19/trackovid-landing-page",
+  "keywords": [
+    "gatsby",
+    "starter",
+    "postcss",
+    "modules",
+    "analytics",
+    "linters"
+  ],
+  "license": "MIT",
+  "main": "n/a",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Tech4covid19/trackovid-landing-page"
+  },
+  "scripts": {
+    "dev": "gatsby develop",
+    "build": "gatsby build",
+    "lint": "eslint src",
+    "lint-styles": "stylelint \"src/**/*.css\" \"src/**/*.scss\""
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
We should pin all packages versions. And to keep dependencies updated, we can add the renovate bot to do that for us 🙂 .

Reply:
Can you do that after this commit?

Reply:
Yes.